clouddatabasesv5/cloud_databases_v5.go

```diff
@@ -4177,6 +4181,10 @@ type GetDefaultScalingGroupsOptions struct {
 	// Database type name.
 	Type *string `json:"type" validate:"required,ne="`
 
+	// When a host_flavor of 'multitenant' is included with the request, IBM Cloud Database's new shared compute default
```

Comment:
Does this mean, only host_flavor 'multitenant' is supported at this point?

Reply:
@obai-1 yes that's correct 😄